components/flow/flow-rest-resource-service/src/main/kotlin/net/corda/flow/rest/FlowStatusCacheService.kt

```diff
@@ -12,7 +12,7 @@ interface FlowStatusCacheService: Lifecycle {
      * Initialises the API implementation. This method may be called multiple times throughout the life
      * of the API.
      */
-    fun initialise(config: SmartConfig)
+    fun initialise(config: Map<String, SmartConfig>)
```

Comment:
Would it not be better:
```suggestion
    fun initialise(messagingConfig: SmartConfig, stateManagerConfig: SmartConfig, restConfig: SmartConfig)
```
?

Reply:
I was on the fence, but I think you're right. It's a more descriptive interface. I'll make that change!